docs/guides/lint.md

```diff
@@ -96,9 +251,18 @@ lint:
       severity: error
 ```
 
-## Additional rule options
+:::warning
+With the short version, you can't configure [additional options](#additional-rule-options) for any given rule (if it supports them).
```

Comment:
```suggestion
With the short configuration syntax, you can't configure [additional options](#additional-rule-options) for any given rule (if it supports them).
```

Reply:
issue: This box should be moved up before we even show the short/verbose examples, so people know why they should use one over the other. Also, moving it up will resolve the problem of having two consecutive admonition boxes